phpunit

```diff
@@ -11,7 +11,7 @@
  */
 
 // Please update when phpunit needs to be reinstalled with fresh deps:
-// Cache-Id-Version: 2016-03-23 14:50 UTC
+// Cache-Id-Version: 2016-03-25 09:44 UTC
```

Comment:
Actually, we do not really need this change here as it is also done in #18306 which is required here to also fix the mocks that are also wrong in the `2.3` branch.